runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -1232,32 +1232,38 @@ def DeleteCommands()
   delcommand Var
   delcommand Winbar
 
-  if exists('k_map_saved')
-    if !empty(k_map_saved) && !k_map_saved.buffer
+  if k_map_saved isnot null_dict
```

Comment:
```suggestion
  if k_map_saved != null
```

Reply:
that is not resolved

Reply:
https://github.com/vim/vim/pull/15013#discussion_r1641661003